Fix: Nested-accordion failing tests - beta [ED-11804]

hein-obox opened the pull request:
## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

davseve committed:
Internal: Changelog v3.15.3  (#23485)

davseve committed:
Internal: Bump main version (#23481)

Commit:
Internal: Fix plugin tester (#23492)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

louiswol94 committed:
Tweak: Ensure downgrade scripts will run [ED-11791] (#23467)

dependabot[bot] committed:
Internal: Bump the elementor-packages group with 4 updates (#23508)

Bumps the elementor-packages group with 4 updates:
[@elementor/editor-app-bar](https://github.com/elementor/elementor-packages/tree/HEAD/packages/core/editor-app-bar),
[@elementor/editor-responsive](https://github.com/elementor/elementor-packages/tree/HEAD/packages/core/editor-responsive),
[@elementor/editor-site-navigation](https://github.com/elementor/elementor-packages/tree/HEAD/packages/core/editor-site-navigation)
and [@elementor/ui](https://github.com/elementor/elementor-ui).

Updates `@elementor/editor-app-bar` from 0.7.1 to 0.7.2
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a
href="https://github.com/elementor/elementor-packages/commits/HEAD/packages/core/editor-app-bar">compare
view</a></li>
</ul>
</details>
<br />

Updates `@elementor/editor-responsive` from 0.6.11 to 0.6.12
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a
href="https://github.com/elementor/elementor-packages/commits/HEAD/packages/core/editor-responsive">compare
view</a></li>
</ul>
</details>
<br />

Updates `@elementor/editor-site-navigation` from 0.18.1 to 0.18.3
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a
href="https://github.com/elementor/elementor-packages/commits/HEAD/packages/core/editor-site-navigation">compare
view</a></li>
</ul>
</details>
<br />

Updates `@elementor/ui` from 1.4.53 to 1.4.54
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a
href="https://github.com/elementor/elementor-ui/commits">compare
view</a></li>
</ul>
</details>
<br />


Dependabot will resolve any conflicts with this PR as long as you don't
alter it yourself. You can also trigger a rebase manually by commenting
`@dependabot rebase`.

[//]: # (dependabot-automerge-start)
[//]: # (dependabot-automerge-end)

---

<details>
<summary>Dependabot commands and options</summary>
<br />

You can trigger Dependabot actions by commenting on this PR:
- `@dependabot rebase` will rebase this PR
- `@dependabot recreate` will recreate this PR, overwriting any edits
that have been made to it
- `@dependabot merge` will merge this PR after your CI passes on it
- `@dependabot squash and merge` will squash and merge this PR after
your CI passes on it
- `@dependabot cancel merge` will cancel a previously requested merge
and block automerging
- `@dependabot reopen` will reopen this PR if it is closed
- `@dependabot close` will close this PR and stop Dependabot recreating
it. You can achieve the same result by closing it manually
- `@dependabot show <dependency name> ignore conditions` will show all
of the ignore conditions of the specified dependency
- `@dependabot ignore <dependency name> major version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's major version (unless you unignore this specific
dependency's major version or upgrade to it yourself)
- `@dependabot ignore <dependency name> minor version` will close this
group update PR and stop Dependabot creating any more for the specific
dependency's minor version (unless you unignore this specific
dependency's minor version or upgrade to it yourself)
- `@dependabot ignore <dependency name> dependency` will close this
group update PR and stop Dependabot creating any more for the specific
dependency (unless you unignore this specific dependency or upgrade to
it yourself)
- `@dependabot unignore <dependency name> dependency` will remove all of
the ignore conditions of the specified dependency
- `@dependabot unignore <dependency name> <ignore condition>` will
remove the ignore condition of the specified dependency and ignore
conditions


</details>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

hein-obox committed:
Internal: Playwright test - nested-accordion are failing [ED-11802] (#23512)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

louiswol94 committed:
Internal: Accurately Convert values from Normal to Custom option in Mobile [ED-11687] (#23516)

StyleShit committed:
Internal: Update dependencies (#23520)

hein-obox committed:
test fix

hein-obox committed:
fix flaky test

hein-obox committed:
Merge branch 'main' into fix/ED-11804-fix-nested-accordion-failing-tests

hein-obox committed:
test fixes

hein-obox committed:
test fixes

hein-obox committed:
Merge remote-tracking branch 'origin/fix/ED-11804-fix-nested-accordion-failing-tests' into fix/ED-11804-fix-nested-accordion-failing-tests

yotamselementor commented on tests/playwright/sanity/modules/nested-accordion/nested-accordion-content.test.ts:
```diff
@@ -150,7 +150,10 @@ test.describe( 'Nested Accordion Content Tests @nested-accordion', () => {
+
+			frame = editor.getPreviewFrame();
+
+			await editor.isUiStable( await frame.locator( '.elementor-edit-mode' ), 10, 2000 );
```
```suggestion
			await editor.isUiStable( frame.locator( '.elementor-edit-mode' ), 10, 2000 );
```

yotamselementor commented on tests/playwright/sanity/modules/nested-accordion/nested-accordion-style.test.ts:
```diff
@@ -236,7 +235,10 @@ test.describe( 'Nested Accordion Style Tests @nested-accordion', () => {
+
+			frame = editor.getPreviewFrame();
+
+			await editor.isUiStable( await frame.locator( '.elementor-edit-mode' ), 10, 2000 );
```
```suggestion
			await editor.isUiStable( frame.locator( '.elementor-edit-mode' ), 10, 2000 );
```

hein-obox committed:
test update

hein-obox committed:
test update

hein-obox committed:
remove test.only()

hein-obox committed:
test improvement

TzviRabinovitch committed:
Internal: Remove minimum installed version for Container [ED-11883] (#23530)

davseve committed:
Merge branch '3.16' into fix/ED-11804-fix-nested-accordion-failing-tests-beta

hein-obox committed:
Fix: Container padding uses incorrect left and right values inside RTL modus [ED-11844] (#23490)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

hein-obox committed:
test update

hein-obox committed:
Merge branch 'main' into fix/ED-11804-fix-nested-accordion-failing-tests

hein-obox committed:
Merge branch 'fix/ED-11804-fix-nested-accordion-failing-tests' into fix/ED-11804-fix-nested-accordion-failing-tests-beta

hein-obox committed:
Merge remote-tracking branch 'origin/fix/ED-11804-fix-nested-accordion-failing-tests-beta' into fix/ED-11804-fix-nested-accordion-failing-tests-beta

hein-obox committed:
Merge branch '3.16' into fix/ED-11804-fix-nested-accordion-failing-tests-beta
